src/service/receive.py

```diff
@@ -0,0 +1,30 @@
+#This File will receive the message from the virtual host and queue and display the message
```

Comment:
Please remove this script from here since it is not a service.

Reply:
Done.
Removed all the comments from all files.

Reply:
Not the comment. The receive.py script.